invenio_app_ils/ui/backoffice/src/pages/ItemDetails/components/ItemMetadata/ItemMetadata.scss

```diff
@@ -2,4 +2,7 @@
   .field-name {
     text-transform: capitalize !important;
```

Comment:
I have a general question: is the usage  of `!important` in css something react specific?

Reply:
Nope! Probably a [Semantic UI React issue](https://react.semantic-ui.com)!

Reply:
@kprzerwa @ntarocco its a semantic issue indeed, the solutions are either styled components, I guess adjustments to the build system so we can configure way css is build, or we keep having our eye sockets bleeding with `!important` crap.